tests/acceptance/fixtures.py

```diff
@@ -301,15 +312,13 @@ def latest_part_image(conversion, mender_image):
 
 
 @pytest.fixture(scope="function")
-def successful_image_update_mender(request, build_image_fn):
+def successful_image_update_mender(request, latest_mender_image):
     """Provide a 'successful_image_update.mender' file in the current directory that
     contains the latest built update."""
 
-    latest_mender_image = latest_build_artifact(build_image_fn(), "core-image*.mender")
-
     shutil.copy(latest_mender_image, "successful_image_update.mender")
 
-    print("Copying 'successful_image_update.mender' to '%s'" % latest_mender_image)
+    print("Copying '%s' to 'successful_image_update.mender'" % latest_mender_image)
```

Comment:
@kacf @merlin-northern This is what is making the vexpress-tests fail.

With the previous code, build_image_fn() returned a yocto build dir with `ssh-server-openssh` (see [here](https://github.com/mendersoftware/meta-mender/blob/d9e6cece4eba175d13606097ea6417528e434606/tests/acceptance/fixtures.py#L372)), while with my change is just returns the original `.mender` that was build.

Then, I assume that for core-image-full-cmdline (i.e. all configurations except vespress-flash) is already included, so this statement does not take any effect.

Finally, why does it fail in `test_signed_updates` and not before? Because the QEMU image being used for tests has openssh server (and scp) so all previous tests using scp go fine, until we arrive to `test_network_based_image_update`, which passes but installs a image witn no scp, and next test to execute is `test_signed_updates`.

Now, I only have to fix (it is not enough with just revert, as this change actually fixed something else in mender-convert). New push is coming up.

Reply:
wow. well done. I was just about to comment that if getting scp in is a problem, then we can reimplement it with i/o redirection, since we have ssh.